core/src/main/java/com/sequenceiq/cloudbreak/service/account/PreferencesService.java

```diff
@@ -57,6 +60,14 @@ public Boolean isPlatformSelectionDisabled() {
         return platforms;
     }
 
+    public Set<String> enabledGovPlatforms() {
+        Set<String> platforms;
+        platforms = enabledGovPlatforms.isEmpty()
```

Comment:
Perhaps replace with `StringUtils.isEmpty(enabledGovPlatforms)` to make it `null`-safe.